add deploy guide (#50)
vu-luong/ezy-smashers: server/README.md

6. Run `console.sh` in `EZYFOX_SERVER_HOME` on your termial, if you want to run `ezyfox-server` in backgroud you will
   need run `start-server.sh` on your terminal

**How to export external libraries to ezyfox server**

1. Move to `EzySmashers-startup` module
2. Run `mvn clean install -Denv.EZYFOX_SERVER_HOME=deploy -Pezyfox-deploy`

> 3. Run class `org.youngmonkeys.ezysmashers.tools.ExternalLibrariesExporter` in `EzySmashers-startup/src/test/java`

Reviewer: This sub-section location is quite confusing. It seems to belong to **Run by ezyfox-server** section

## Run without ezyfox-server

> To run without `ezyfox-server` you need follow by steps:

Reviewer: ```suggestion
To run without `ezyfox-server`, you can follow the steps outlined below:
```

1. Run `bash export.sh` command (or `\\.export.bat` on Windows)
2. Move to `EzySmashers-startup/deploy` folder

### On Windows

You just need run `console.bat`

### On Linux

> 1. To run to debug, you just need run `bash console.sh` on your terminal
> 2. To run in background, you just need run `bash start-service.sh` on your terminal
> 3. To stop your service, you just need run `bash stop-service.sh` on your terminal

Reviewer: ```suggestion
1. For debugging purposes, run `bash console.sh` in your terminal
2. To run in background, execute `bash start-service.sh` in your terminal
3. To stop the service, execute the script `bash stop-service.sh` in your terminal
```

## Run with specific configuration profile

You can [read this guide](https://youngmonkeys.org/ezyfox-server-project-configuration/) to know how to
run `ezyfox-server` or your application with a specific profile

# How to deploy?

> You can take a look this guide: [Deploy EzyFox Server](https://youngmonkeys.org/deploy-ezyfox-server/)

Reviewer: ```suggestion
You can take a look at this guide: [Deploy EzyFox Server](https://youngmonkeys.org/deploy-ezyfox-server/)
```

3. EzySmashers-common => `ezyfox-server/common/ EzySmashers-common-1.0.0.jar`
4. EzySmashers-plugin => `ezyfox-server/plugins/EzySmashers-plugin`

## Deploy with tools

> You can use bellow tools to copy jar files (follow by above mapping)

Reviewer: ```suggestion
You can use bellow tools to copy jar files (refer to the above mapping)
```

1. Move to `EzySmashers-startup` module
2. Run `ApplicationStartup` in `src/main/java`
3. Run `ClientTest` in `src/test/java`

# Documentation

> You can find a lot of documents on [youngmonkeys.org](https://youngmonkeys.org/ezyfox-sever/)

Reviewer: ```suggestion
Additional documentation is available at [youngmonkeys.org](https://youngmonkeys.org/ezyfox-sever/)
```
